src/TwigComponent/src/Twig/SlotNode.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\TwigComponent\Twig;
+
+use Twig\Compiler;
+use Twig\Node\Expression\AbstractExpression;
+use Twig\Node\Node;
+use Twig\Node\NodeOutputInterface;
+
+/**
+ * @author Mathéo Daninos <matheo.daninos@gmail.com>
+ *
```

Comment:
If these classes are taken from that other library, I’d love a link to the original implementation… it just feels better to give some proper credit